Managers/UInputMan.cpp

```diff
@@ -7,6 +7,8 @@
 #include "PerformanceMan.h"
 #include "GUIInput.h"
```

Comment:
I'm not 100% sure of this, but I would guess that BuyMenuGUI.h includes GUIInput.h in some chain of headers. Try removing this include and seeing if it'll still compile.

Reply:
It threw a fit here, and from what I can tell it doesn't include it.